build.xml

```diff
@@ -1042,6 +1042,8 @@
         <source path="${plugins.dir}/index-geoip/src/java/" />
         <source path="${plugins.dir}/index-links/src/java/" />
         <source path="${plugins.dir}/index-links/src/test/" />
+        <source path="${plugins.dir}/index-jexl-filter/src/java/" />
```

Comment:
Can you also please make the entry for JavaDoc. You will find the entries elsewhere in build.xml

Reply:
- the index-jexl-filter dir should also be added to the Eclipse target: there will be 4 entries in the main build.xml
- the plugin folders in build.xml are sorted lexicographically: index-jexl-filter should come before index-links
- please also add the package to plugins.index in default.properties. That's required to list the pluging on the [Javadoc overview page](https://builds.apache.org/job/nutch-trunk/javadoc/)
- also a package-info.java would be great

Reply:
@sebastian-nagel regarding default.properties: Is that the package name? Because the package name is org.apache.nutch.indexer.filter, which is already there. Should I change the package name?

Reply:
Yes, of course, the package name should be descriptive, e.g. org.apache.nutch.indexer.jexl. Thanks!